### ISSUE TITLE
Make radio chat include heads of staff's titles (CMO, CE etc.)

### ISSUE THREAD

**Issue by Hreno** [space-wizards/space-station-14#32010]
## Description

Given that every round changes roles, it's hard too keep up with who the captain is, who head of security, etc.

It'd be helpful to have titles of important persons (heads of staff) in the chatbox; maybe in the form of an acronym or abbreviation. So if John Doe is a CMO, the chat would say perhaps:
```
John Doe (CMO): Where did the chemist go?
```

Why only heads of staff? Because, in-character, everyone on the station would likely be familiar with them and their voices but not necessarily with other employees on the station.

**Comment by lzk228** [comment]
as i remember, i ss13 there was a telecomms console or something with some settings
and there you could turn on bold for heads, turn on job displaying and type of it (`Name, Job`, `Name (Job)`, etc.)

**Comment by Hreno** [comment]
Well it could be done for every job as well. A drawback comes to mind though: stealth infiltrator type of antags would be recognized more quickly.

**Comment by slarticodefast** [comment]
Yeah, that would kinda make the voice mask useless for coms unless you also have an agent ID.

**Comment by Cojoke-dot** [comment]
> Yeah, that would kinda make the voice mask useless for coms unless you also have an agent ID.

Maybe it could use the manifest and base it on whoever is listed as a head on there?

**Comment by deltanedas** [comment]
dynamic per player identities wyci
